TheMovieDb/Reviews/Controller/ReviewsCollectionViewController.swift

```diff
@@ -0,0 +1,103 @@
+//
+//  ReviewsCollectionViewController.swift
+//  TheMovieDb
+//
+//  Created by Javier Cueto on 31/10/21.
+//
+
+import UIKit
+
+private let reuseIdentifier = "Cell"
+
+final class ReviewsCollectionViewController: UICollectionViewController {
+    
+    // MARK: - Properties
+    public var movie: Movie
+    private var reviews = [Review]()
+    
+    // MARK: - LifeCycle
+    init(with movie: Movie) {
+        self.movie = movie
+        let layout = UICollectionViewFlowLayout()
+        layout.sectionInset = .init(top: 0, left: 10, bottom: 0, right: 10)
+        super.init(collectionViewLayout: layout)
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureUI()
+        fetchDataAPI()
+    }
+
+    // MARK: - Helpers
+    
+    private func configureUI() {
+        self.collectionView!.register(ReviewCell.self, forCellWithReuseIdentifier: ReviewCell.reuseIdentifier )
+        navigationItem.title = "Reviews"
+        collectionView.backgroundColor = .systemBackground
+    }
+    
+    // MARK: - API
+    
+    private func fetchDataAPI() {
+        let id = String(movie.id)
+        let parameter = APIParameters(id: id)
+        MovieAPI.shared.fetchData(endPoint: .review, with: parameter, completion: {(response: Result<Reviews, Error>) in
+            switch response {
+            case .failure(let error):
+                debugPrint(error)
+            case .success(let res):
+                self.reviews = res.reviews
+                DispatchQueue.main.async {
+                    self.collectionView.reloadData()
+                }
+            }
+       
+        })
+    }
+}
+
+// MARK: UICollectionViewDataSource
+extension ReviewsCollectionViewController {
+    override func numberOfSections(in collectionView: UICollectionView) -> Int {
+        // #warning Incomplete implementation, return the number of sections
+        return 1
+    }
```

Comment:
if a single section is handled you can remove this override

Reply:
I already removed that part, the code works  😅. Thanks